README.md

```diff
@@ -0,0 +1,207 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Comming soon</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">Comming soon</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Github Flow]**
+- **[React]**
+- **[Tailwindcss]**
+- **[Jest]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo](https://ander-pokemons.netlify.app)
+- [Video Presentation](https://drive.google.com/file/d/18Ul3HXNyjkL1f9-Iws3YLOu_j8FLY9dk/view?usp=share_link)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to :
+Fork this project
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd foldername
+- git clone https://github.com/anderCM/Metrics.git
+
+### Install
+
+Install this project with:
+
+Fork this project
+
+### Usage
+
+To run the project, execute:
+- npm install
+- npm run build
+- npm run dev
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm run test
+
+### Deployment
+
+You can deploy this project using:
+
+- Netlify
+- Vercel
+- Heroku
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Anderson**
+
+- GitHub: [@anderCM](https://github.com/anderCM)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andersoncanales/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Tests]**
+- [ ] **[Fix bugs]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a coffee
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank my team
+
+- Original idea: Nelson Sakwa on Behance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[When will new features will be available?]**
+
+  - [In the first quarter of 2023]
+
+- **[When will you fix bugs?]**
+
+  - [When we find the way]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job @anderCM 🤝

- [ ] [Optional] Kindly fix this broken MIT license link to make your readme file more professional.

Reply:
Thanks, I am adding it